src/vcpkg/install.cpp

```diff
@@ -805,10 +791,16 @@ namespace vcpkg::Install
                     {
                         auto omitted = targets.size() - 4;
                         library_target_pair.second.erase(targets.begin() + 4, targets.end());
-                        msg.append(
-                            msg::format(msgCmakeTargetsExcluded, msg::count = omitted).appendnl().extract_data());
+                        msg.append(LocalizedString()
+                                       .append_indent()
+                                       .append_raw("# ")
+                                       .append(msgCmakeTargetsExcluded, msg::count = omitted)
+                                       .extract_data());
```

Comment:
We should get rid of this until `msg` is actually a properly localized message.

Reply:
Yeah I think I'm going to wontfix this one for now, sorry.